src/ucp/core/ucp_worker.c

```diff
@@ -3128,7 +3131,7 @@ static void ucp_worker_discard_tl_uct_ep(ucp_ep_h ucp_ep, uct_ep_h uct_ep,
     req->send.ep                            = ucp_ep;
     req->send.uct.func                      = ucp_worker_discard_uct_ep_pending_cb;
     req->send.state.uct_comp.func           = ucp_worker_discard_uct_ep_flush_comp;
-    req->send.state.uct_comp.count          = 1;
+    req->send.state.uct_comp.count          = 0;
```

Comment:
should we check/assert in line 2427 that cut_comp.count = 0? and call ucp_worker_discard_uct_ep_flush_comp